README.md

```diff
@@ -140,6 +150,6 @@ Definitely! We welcome any improvements if your changes are good and align with
 
 ## 📝 License <a name="license"></a>
 
-This project is [MIT](./MIT.md) licensed.
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] As indicated by the previous reviewer, your MIT license link is still broken and leads to error page 404. Kindly fix it.
![Screen Shot 2023-10-25 at 12 20 08 AM](https://github.com/kayjeee/hello-rails-backend/assets/62849872/68f11d67-b5a8-4b32-aaa7-55d1a877ab4b)

Reply:
fixed